core/src/test/java/nu/yona/server/entities/MockJpaRepositoryEntityWithUuid.java

```diff
@@ -106,12 +107,7 @@ public void deleteAllInBatch()
 	@Override
 	public T getOne(UUID id)
```

Comment:
Is this still called `getOne`?

Reply:
Indeed. ``JpaRepository`` has ``getOne``.